### PR TITLE
Feature/jjs/add sinatra

### PR DESCRIPTION
**Pull request by jordanskomer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is a derivative of [Keep a Changelog](http://keepachangelog.com/).
+
+## 2017-03-31
+### Added
+- Sinatra. @jordanskomer
+
```
Comment (jordanskomer) [inline CHANGELOG.md]: Could you add your entry into the changelog?

**Comment by jordanskomer** [comment]
Sure thing

**Comment by jordanskomer** [comment]
image

File: CHANGELOG.md (hunk 1 above)
Comment (jordanskomer) [inline CHANGELOG.md]: image

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Change Log
```
Comment (jordanskomer) [inline CHANGELOG.md:1]: Can you update the changelog?

Comment (jordanskomer) [reply CHANGELOG.md:1]: image

**Comment by jordanskomer** [comment]
Evan would you say this changes your perspective?

**Comment by jordanskomer** [comment]
image

**Comment by EvanAgee** [comment]
Testing to see if changling or changelog will trigger

**Comment by jordanskomer** [comment]
image

**Comment by jordanskomer** [comment]
chang

**Comment by anniemations** [comment]
image

**Comment by anniemations** [comment]
chang

**Comment by anniemations** [comment]
image

**Comment by anniemations** [comment]
image

File: Gemfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
+gem "sinatra"
+gem "dotenv"
+gem "haml"
+gem "json"
+gem "rest-client"
```
Comment (jordanskomer) [inline Gemfile:7]: garbage.

**Pull request merged**